### ISSUE TITLE
Allow setting a default sbt plugin version

### ISSUE THREAD

**Issue by jroper** [sbt/sbt#3444]
sbt cross building should allow setting a default sbt plugin version other than the current version of sbt. This is particularly useful when importing into IDEs, as it will allow you to import a sbt 0.13 build using an sbt 1.0 classpath for the plugin, so that you can actually work on upgrading.

Currently you can do that by adding this to your `build.sbt`:

```scala
sbtVersion in pluginCrossBuild := "1.0.0"
scalaVersion := "2.12.2"
```

However, you can't leave it in there, as that completely breaks cross building, with the above config, the cross building support fails to correctly switch to `0.13`.

**Comment by eed3si9n** [comment]
So you want something like `sbtVersion in pluginCrossBuild`, but explicitly weaker than `sbtVersion in pluginCrossBuild` so `^^0.13.16` still works?

**Comment by jroper** [comment]
Yes, but note that I don't think it has to be weaker, what seems to be the problem here is just a matter of ordering of settings.  Unless it's a matter of scopes... I wonder if the following works:

```scala
sbtVersion in pluginCrossBuild in ThisBuild := "1.0.0"
scalaVersion in ThisBuild := "2.12.2"
```

**Comment by jroper** [comment]
`in ThisBuild` makes no difference.

**Comment by dwijnand** [comment]
I've been setting `sbtVersion in Global` so that I may use the latest version of sbt for the build but allow my plugin to be used with earlier versions:

https://github.com/dwijnand/sbt-travisci/blob/c3314e93bef919d47315301c7191e40ef94bd5d3/build.sbt#L13
